ui/app/helpers/format-month-ts.js

```diff
@@ -1,8 +1,8 @@
 import moment from 'moment';
 import Helper from '@ember/component/helper';
 
-export function formatMonthTs([date]) {
-  const format = 'MMM DD HH:mm:ss ZZ';
+export function formatMonthTs([date], options = {}) {
+  const format = options.short ? 'MMM DD' : 'MMM DD HH:mm:ss ZZ';
```

Comment:
Could use `D` instead of `DD` for a slightly shorter date in early parts of the month? But maybe the leading zero is desired.

Reply:
Ah, good call.